language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/ContextTypeResolver.java

```diff
@@ -258,15 +258,21 @@ public ContextTypeResolver(PositionedOperationContext context) {
             Predicate<Symbol> predicate = symbol -> symbol.getName().isPresent()
                     && symbol.kind() == SymbolKind.FUNCTION;
             Predicate<Symbol> qNamePredicate =
-                    predicate.and(symbol -> symbol.getName().get().equals(qNameRef.identifier().text()));
+                    predicate.and(symbol -> symbol.getName().isPresent()
+                            && symbol.getName().get().equals(qNameRef.identifier().text()));
             funcSymbol = this.getTypeFromQNameReference(qNameRef, qNamePredicate);
         } else if (nameRef.kind() == SyntaxKind.SIMPLE_NAME_REFERENCE) {
             funcSymbol = getSymbolByName(((SimpleNameReferenceNode) nameRef).name().text());
         } else {
             return Optional.empty();
         }
```

Comment:
Remove this else clause now. Line 270 satisfies the empty return

Reply:
Done!